ui-v2/ember-cli-build.js

```diff
@@ -2,7 +2,19 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 module.exports = function(defaults) {
-  let app = new EmberApp(defaults, {
+  const env = EmberApp.env();
+  const prodlike = ['production', 'staging'];
+  const isProd = env === 'production';
+  const isProdLike = prodlike.indexOf(env) > -1;
+  const sourcemaps = !isProd;
+  let app = new EmberApp(
+    Object.assign(
+      {},
+      defaults,
+      {
+        productionEnvironments: prodlike
```

Comment:
Just curious -  are you're planning on using this other places in the app?

Reply:
Sorry @meirish ! Another one I've forgotten to come back to you on! When I first started on this I was toying with the performance testing idea thing, which would have meant being able to have like a 'staging' build that was the same as 'prod' but with the performance things in.

Pretty sure I don't need this as I'm doing a lot of that in my testing layer now, but I came across the `productionEnvironments` whilst looking into enabling CSS sourcemaps, so added it in here also in case I start putting any performance monitoring in the build as well as the tests. If I do that means I can keep the perf. things in for running on my preview builds whilst keeping them out for the actual production builds.